### DISCUSSION TITLE
Should images need to have a session id to access them

### DISCUSSION THREAD

**Discussion by Fletcher-Alderton** [go-shiori/shiori#708] [Q&A]
I was playing with the api and noticed that you don’t need session id in the header to access images.
I don’t think its that much of an issue as its unlikely some one is bookmarking something not available on the web but they might bookmark something like a photo.

**Comment by fmartingr** [comment]
Ummm this raises a good point and I don't know if I'm checking this in the new endpoints to be honest, severity shouldn't be high but still something to be addressed. Moving this to an issue for review. Thanks for the report.
